client/accounting.go

```diff
@@ -2,82 +2,102 @@ package client
 
 import (
 	"context"
-	"fmt"
 
 	v2accounting "github.com/nspcc-dev/neofs-api-go/v2/accounting"
 	rpcapi "github.com/nspcc-dev/neofs-api-go/v2/rpc"
 	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
-	v2signature "github.com/nspcc-dev/neofs-api-go/v2/signature"
 	"github.com/nspcc-dev/neofs-sdk-go/accounting"
 	"github.com/nspcc-dev/neofs-sdk-go/owner"
 )
 
-type BalanceOfRes struct {
+// GetBalancePrm groups parameters of GetBalance operation.
+type GetBalancePrm struct {
+	ownerSet bool
+	ownerID  owner.ID
+}
+
+// SetAccount sets identifier of the NeoFS account for which the balance is requested.
+// Required parameter. Must be a valid ID according to NeoFS API protocol.
+func (x *GetBalancePrm) SetAccount(id owner.ID) {
+	x.ownerID = id
+	x.ownerSet = true
+}
+
+// GetBalanceRes groups resulting values of GetBalance operation.
+type GetBalanceRes struct {
 	statusRes
 
 	amount *accounting.Decimal
 }
 
-func (x *BalanceOfRes) setAmount(v *accounting.Decimal) {
+func (x *GetBalanceRes) setAmount(v *accounting.Decimal) {
 	x.amount = v
 }
 
-func (x BalanceOfRes) Amount() *accounting.Decimal {
+// Amount returns current amount of funds on the NeoFS account as decimal number.
+//
+// Client doesn't retain value so modification is safe.
+func (x GetBalanceRes) Amount() *accounting.Decimal {
 	return x.amount
 }
 
-// GetBalance receives owner balance through NeoFS API call.
+// GetBalance requests current balance of the NeoFS account.
 //
 // Any client's internal or transport errors are returned as `error`,
 // If WithNeoFSErrorParsing option has been provided, unsuccessful
 // NeoFS status codes are returned as `error`, otherwise, are included
 // in the returned result structure.
-func (c *Client) GetBalance(ctx context.Context, owner *owner.ID, opts ...CallOption) (*BalanceOfRes, error) {
-	// apply all available options
-	callOptions := c.defaultCallOptions()
-
-	for i := range opts {
-		opts[i](callOptions)
+//
+// Immediately panics if parameters are set incorrectly (see GetBalancePrm docs).
+// Context is required and must not be nil. It is used for network communication.
+//
+// Exactly one return value is non-nil. Server status return is returned in GetBalanceRes.
```

Comment:
it conflicts a little with the paragraph above (that has info about `WithNeoFSErrorParsing`)